### DISCUSSION TITLE
Unable to unsubscribe from update emails

### DISCUSSION THREAD

**Discussion by recy21** [community/community#52313] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hey,

My colleague lost his password and 2fa.
He can't unlink this email, as this would need at least a password.
He can't reset his password, as 2fa is necessary to do that.
But he is spammed by all project he subscribed all day long.
Clicking on the unsubscribe link doesn't solve the problem, because you need to be logged in to unsubscribe.

My question: Why? GitHub seems the only place in the internet you need to be logged in to unsubscribe.
It can't be that blocking Github emails is the only option here.

Thanks,
Recy

**Comment by SMelidoni** [comment, marked as answer]
Hi @recy21 

It seems like your colleague is in a difficult situation. Unfortunately, GitHub requires both the password and 2FA authentication for security reasons before any account-related actions can be taken. This includes unsubscribing from email notifications.

In this case, I would recommend your colleague to contact GitHub Support directly to explain the situation and see if they can provide any assistance in recovering the account or at least help with unsubscribing from the email notifications. Here's the link to contact GitHub Support: https://support.github.com/contact

Meanwhile, as a temporary solution, your colleague could create email filters to automatically move GitHub notifications to a specific folder or mark them as read to avoid getting spammed.

**Reply by recy21** [reply]
Thanks, will try the Support, wasn't able to find the form in the first place, kinda tricky

**Comment by Hot-Amir** [comment]
> ### Select Topic Area
> Question
> 
> ### Body
> Hey,
> 
> My colleague lost his password and 2fa. He can't unlink this email, as this would need at least a password. He can't reset his password, as 2fa is necessary to do that. But he is spammed by all project he subscribed all day long. Clicking on the unsubscribe link doesn't solve the problem, because you need to be logged in to unsubscribe.
> 
> My question: Why? GitHub seems the only place in the internet you need to be logged in to unsubscribe. It can't be that blocking Github emails is the only option here.
> 
> Thanks, Recy
